smack-extensions/src/main/java/org/jivesoftware/smackx/ping/PingManager.java

```diff
@@ -419,21 +419,24 @@ public synchronized void pingServerIfNecessary() {
             // Ping has been disabled
             return;
         }
-        long lastStanzaReceived = connection.getLastStanzaReceived();
-        if (lastStanzaReceived > 0) {
-            long now = System.currentTimeMillis();
-            // Delta since the last stanza was received
-            int deltaInSeconds = (int)  ((now - lastStanzaReceived) / 1000);
-            // If the delta is small then the ping interval, then we can defer the ping
-            if (deltaInSeconds < pingInterval) {
-                maybeSchedulePingServerTask(deltaInSeconds);
-                return;
-            }
-        }
+       
         if (connection.isAuthenticated()) {
             boolean res = false;
 
             for (int i = 0; i < TRIES; i++) {
+            	{// check if there was an package meanwhile, then we don't need a ping...
```

Comment:
That comment should probably go into the next line. Also there might be one too many parenthesis here?
Also that line is indented incorrectly.